sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/internal/impl/DefaultDBBasedGeoVelocityDataResolver.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.extension.siddhi.execution.geovelocity.internal.impl;
+
+import org.wso2.extension.siddhi.execution.geovelocity.api.GeoVelocityData;
+import org.wso2.extension.siddhi.execution.geovelocity.api.GeoVelocityDataResolver;
+import org.wso2.extension.siddhi.execution.geovelocity.internal.exception.GeoVelocityException;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+
+/**
+ * The default implementation of the GeoVelocityResolver interface. This is implemented based on RDBMS.
+ */
+public class DefaultDBBasedGeoVelocityDataResolver implements GeoVelocityDataResolver {
+
+    @Override
+    public void init(ConfigReader configReader) throws GeoVelocityException {
+        RDBMSGeoVelocityDataResolver.getInstance().init(configReader);
+    }
+
+    /**
+     * Calls external system or database database to find the geovelocity data.
+     * Can be used by an extended class.
+     * @param username username
```

Comment:
A new line here to separate description and the parameters. Check all the doc comments